lib/install/config/webpacker.yml

```diff
@@ -33,7 +33,7 @@ development:
     host: localhost
     port: 8080
     https: false
-    hot_reloading: false
+    hmr: false
```

Comment:
@ytbryan I agree in the sense that `DevServer.hot?` and `WEBPACKER_HOT` are a bit ambiguous.